.gitignore

```diff
@@ -23,3 +23,6 @@ yarn-debug.log*
 yarn-error.log*
 .reshuffle*
 .env
+
```

Comment:
This seems like the wrong place for this - it should be `~/.gitignore`, not inside a repo.